### PR TITLE
Revert "Added in gitignore"

### PR DESCRIPTION
**Pull request by briangonzalez**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/briangonzalez/termrc/pull/19" target="_blank">ReviewNinja

File: .gitignore [hunk 1]
```diff
@@ -1,4 +0,0 @@
-.DS_store
-
```
Comment (briangonzalez) [inline .gitignore]: !fix Please also add in bower_components

Comment (briangonzalez) [inline .gitignore]: !resolved How does it look now?

Comment (briangonzalez) [inline .gitignore]: !resolved how does it look now?

**Comment by briangonzalez** [comment]
!star

**Comment by briangonzalez** [comment]
!unstar

**Comment by briangonzalez** [comment]
:+1:

**Comment by briangonzalez** [comment]
:-1:

**Comment by briangonzalez** [comment]
Project moved and rewritten here: https://github.com/eggplanetio/mert

**Pull request closed without merging**